src/components/search-box/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import classNames from '@components/search-box/searchbox.module.scss';
+import { searchMemberContext } from '@store/search-members/searchMembers-context';
+
+const SearchBox = () => {
+  const { setSearchTerm } = searchMemberContext();
+
+  return (
+    <div className={classNames.searchbox_container}>
+      <input
+        className={classNames.searchbox}
+        type="text"
+        placeholder="Search by Skills, Accomplishments levels, Name"
+        onChange={(e) => setSearchTerm(e.target.value)}
```

Comment:
We have a submit button, so is there any need to have onChange function?

Reply:
Yes after this PR I need to work on a modal where we will be filtering the members according to the skills , for that we have a button .